### DISCUSSION TITLE
I want help in reversing columns in row when viewed on Mobile

### DISCUSSION THREAD

**Discussion by VuppalaMahesh** [fomantic/Fomantic-UI#1477] [Collections (form, grid, menu, message, table, ...)] [answered]
# Help Wanted
I have two rows with combination of text and image combination columns.
1st row:
1st column: text
2nd column: Image

2nd row:
1st column: Image
2nd column:text

So, whenever I see on Mobile, only the 2nd row columns should be reversed(like text should come in 1st column and image should come in 2nd column). I have tried .ui.mobile.reversed property but couldn't succeed. Attaching referencing images and link.

## Testcase
https://jsfiddle.net/aptgvbou/
![CE19649C-C977-4C37-96E6-F50BFBCFD65A](https://user-images.githubusercontent.com/51159897/82647637-20c16880-9c34-11ea-9ffc-df02bb70c8a7.jpeg)
![FFCE71F7-DA77-42FA-9DD0-EECA4998A101_4_5005_c](https://user-images.githubusercontent.com/51159897/82647643-23bc5900-9c34-11ea-9db6-e10a849a0bef.jpeg)

**Comment by exoego** [comment]
Try `mobile reversed` variation.
https://fomantic-ui.com/collections/grid.html#reverse-order

**Comment by VuppalaMahesh** [comment]
Hi exoego, 
Thanks for your quick repsonse. I tried that initially, what I am looking is. Whenever I switch to Mobile version, columns of a row should display one below the other(for which I used stackable). I want only second row columns should be reversed like, first text and image later. First row column elements should remain as it is. Help me with this please. Thanks in advance.

**Comment by exoego** [comment, marked as answer]
I made a demo https://jsfiddle.net/bscmv0ro/
I think `.row` wrapper is not needed, since `two column` is explicitly specified.
Does this demo satisfies your needs ?

**Comment by VuppalaMahesh** [comment]
@exoego . Yes, you are correct. .row wrapper is not needed. Once I removed it, worked fine by adding mobile reversed. You are really quick and saved my time. 
I would like to appreciate this community for helping in such a manner that Newbies would get encouragement to move further. Once again special Thanks to exoego.

I would like to ask, if do we have any website code to practise, how to code exactly by pre-defined Semantic UI classes.

**Comment by exoego** [comment]
Closing since the initial usage question was resolved.

> I would like to ask, if do we have any website code to practise, how to code exactly by pre-defined Semantic UI classes.

Hmm, sorry, but I do not know any external websites or books for studying Semantic-UI/Fomantic-UI...

I and my team always learn from their official document websites.
IMHO, those official docs (especieally Fomantic-UI's) offer extensive guides and examples, up-to-date, so we can learn quickly.

Feel free to open issues and send patchs (pull requests) to documents, if you find rooms for improvement !!
